src/sentry/api/issue_search.py

```diff
@@ -22,77 +14,40 @@
-    key_mappings = {
-        "assigned_to": ["assigned"],
-        "bookmarked_by": ["bookmarks"],
-        "subscribed_by": ["subscribed"],
-        "assigned_or_suggested": ["assigned_or_suggested"],
-        "first_release": ["first-release", "firstRelease"],
-        "first_seen": ["age", "firstSeen"],
-        "last_seen": ["lastSeen"],
-        "active_at": ["activeSince"],
-        # TODO: Special case this in the backends, since they currently rely
-        # on date_from and date_to explicitly
-        "date": ["event.timestamp"],
-        "times_seen": ["timesSeen"],
-        "sentry:dist": ["dist"],
-    }
-    numeric_keys = SearchVisitor.numeric_keys.union(["times_seen"])
-    date_keys = SearchVisitor.date_keys.union(["active_at", "date"])
-
-    @cached_property
-    def is_filter_translators(self):
-        is_filter_translators = {
-            "assigned": (SearchKey("unassigned"), SearchValue(False)),
-            "unassigned": (SearchKey("unassigned"), SearchValue(True)),
-            "for_review": (SearchKey("for_review"), SearchValue(True)),
-            "linked": (SearchKey("linked"), SearchValue(True)),
-            "unlinked": (SearchKey("linked"), SearchValue(False)),
-        }
-        for status_key, status_value in STATUS_QUERY_CHOICES.items():
-            is_filter_translators[status_key] = (SearchKey("status"), SearchValue(status_value))
-        return is_filter_translators
-
-    def visit_is_filter(self, node, children):
-        # the key is "is" here, which we don't need
-        negation, _, _, _, search_value = children
-
-        if search_value.raw_value.startswith("["):
-            raise InvalidSearchQuery('"in" syntax invalid for "is" search')
-
-        if search_value.raw_value not in self.is_filter_translators:
-            raise InvalidSearchQuery(
-                'Invalid value for "is" search, valid values are {}'.format(
-                    sorted(self.is_filter_translators.keys())
-                )
-            )
-
-        search_key, search_value = self.is_filter_translators[search_value.raw_value]
-
-        operator = "!=" if is_negated(negation) else "="
-
-        return SearchFilter(search_key, operator, search_value)
-
-    def visit_boolean_operator(self, node, children):
-        raise InvalidSearchQuery(
-            'Boolean statements containing "OR" or "AND" are not supported in this search'
-        )
+is_filter_translation = {
+    "assigned": ("unassigned", False),
+    "unassigned": ("unassigned", True),
+    "for_review": ("for_review", True),
+    "linked": ("linked", True),
+    "unlinked": ("linked", False),
+}
+for status_key, status_value in STATUS_QUERY_CHOICES.items():
+    is_filter_translation[status_key] = ("status", status_value)
+
+
+issue_search_config = SearchConfig(**default_config.__dict__)
+issue_search_config.allow_boolean = False
+issue_search_config.is_filter_translation = is_filter_translation
+issue_search_config.numeric_keys |= {"times_seen"}
```

Comment:
This will modify the `numeric_keys` on event_search. We're probably better off doing `issue_search_config.numeric_keys = {"times_seen"} | issue_search_config.numeric_keys`, or creating the config safely above.

```
In [1]: from sentry.api.event_search import default_config

In [2]: default_config.numeric_keys
Out[2]:
{'apdex',
 'count_miserable',
 'count_miserable_new',
 'failure_rate',
 'issue.id',
 'p75',
 'p95',
 'p99',
 'project.id',
 'project_id',
 'stack.colno',
 'stack.lineno',
 'stack.stack_level',
 'transaction.duration',
 'user_miser_new',
 'user_misery'}

In [3]: from sentry.api.issue_search import issue_search_config

In [4]: default_config.numeric_keys
Out[4]:
{'apdex',
 'count_miserable',
 'count_miserable_new',
 'failure_rate',
 'issue.id',
 'p75',
 'p95',
 'p99',
 'project.id',
 'project_id',
 'stack.colno',
 'stack.lineno',
 'stack.stack_level',
 'times_seen',
 'transaction.duration',
 'user_miser_new',
 'user_misery'}
 ```

Reply:
Thanks for checking me

Reply:
Ok swapped out the `__dict__` for a call to `asdict` and now we're good

```
In [4]: default_config.numeric_keys
Out[4]:
{'apdex',
 'count_miserable',
 'count_miserable_new',
 'failure_rate',
 'issue.id',
 'p75',
 'p95',
 'p99',
 'project.id',
 'project_id',
 'stack.colno',
 'stack.lineno',
 'stack.stack_level',
 'transaction.duration',
 'user_miser_new',
 'user_misery'}

In [5]: from sentry.api.issue_search import issue_search_config

In [6]: default_config.numeric_keys
Out[6]:
{'apdex',
 'count_miserable',
 'count_miserable_new',
 'failure_rate',
 'issue.id',
 'p75',
 'p95',
 'p99',
 'project.id',
 'project_id',
 'stack.colno',
 'stack.lineno',
 'stack.stack_level',
 'transaction.duration',
 'user_miser_new',
 'user_misery'}

In [7]: issue_search_config.numeric_keys
Out[7]:
{'apdex',
 'count_miserable',
 'count_miserable_new',
 'failure_rate',
 'issue.id',
 'p75',
 'p95',
 'p99',
 'project.id',
 'project_id',
 'stack.colno',
 'stack.lineno',
 'stack.stack_level',
 'times_seen',
 'transaction.duration',
 'user_miser_new',
 'user_misery'}
```